packages/mint-components/src/components/tax-and-cash/sqm-banking-info-form/sqm-banking-info-form.feature

```diff
@@ -110,3 +110,13 @@ Feature: Banking Information Form
     Then a request is made to fetch the form data
     But the request fails
     Then a loading error banner appears
+
+  @motivating
+  Scenario: Editting withdrawal settings requires a 2FA code
```

Comment:
What happens if they make a change then go right back and make another? Is their sessions still "verified" or do they have to complete the 2FA flow again?

Reply:
No session management in the frontend so they would have to go through the 2FA flow every time they try to update their withdrawal settings